packages/component-submission/client/pages/SubmissionWizard.js

```diff
@@ -50,15 +50,15 @@ const SubmissionWizard = ({
           handleButtonClick={updateManuscript}
           history={history}
           initialValues={initialValues}
-          nextUrl={`${match.url}/submission`}
+          nextUrl={`${match.url}/details`}
           previousUrl={`${match.url}/author`}
           step={1}
           validationSchema={filesSchema}
         />
       )}
     />
     <TrackedRoute
-      path={`${match.path}/submission`}
+      path={`${match.path}/details`}
```

Comment:
@hdrury1 I've changed the url for the details page to be consistent. On the off chance someone has bookmarked that page, they should fall throught to the redirect  https://github.com/elifesciences/elife-xpub/blob/f4e3f01640c504dc3813a2474a095d53c9d138e8/packages/component-submission/client/pages/SubmissionWizard.js#L128 and be taken to the author step.